cranelift/filetests/filetests/isa/x64/simd-arithmetic-run.clif

```diff
@@ -0,0 +1,280 @@
+test run
```

Comment:
👍 for bringing the other tests in as well; we can uncomment the others as we get to them. I do think we should add `compile` tests for `fmin/fmax` like I did in `simd-lane-access-compile.clif`; that way we test that the sequence emitted stays correct over time. I don't think we should do it for simpler stuff but for long, tricky sequences like these I feel it would be helpful.